src/crypto/CHIPCryptoPALOpenSSL.cpp

```diff
@@ -104,7 +104,8 @@ static void _logSSLError()
         const char * err_str_reason  = ERR_reason_error_string(ssl_err_code);
         if (err_str_lib)
         {
-            ChipLogError(Crypto, " ssl err  %s %s %s\n", err_str_lib, err_str_routine, err_str_reason);
+            ChipLogError(Crypto, " ssl err  %s %s %s\n", StringOrNullMarker(err_str_lib), StringOrNullMarker(err_str_routine),
```

Comment:
`err_str_lib` is known to be non-null here, but I guess this is OK from a "people might copy/paste this" perspective.